portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,79 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body,
+html {
+    margin: 0;
+    padding: 0;
+}
+.jumbotron {
+    align-items: center;
+    display: flex;
+    background-image: linear-gradient(rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url("/images/jumboBack.jpg");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 50vh;
+    justify-content: center;
+    position: relative;
+}
```

Comment:
FYI typically we sort css properties alphabetically (many IDEs have a sort function that can do it for you).
Makes things less confusing to refactor later

Reply:
Okay I was doing this, might have missed a few cases like "display". Does this apply to the entire CSS file as well, should the classes/id be in alphabetical order?

Reply:
Just the individual attributes should be in alphabetical order. Classes/ids/etc can be however makes the best logical sense.